src/Wallabag/ApiBundle/Controller/EntryRestController.php

```diff
@@ -558,124 +558,81 @@ public function postEntriesListAction(Request $request, EntryRepository $entryRe
      * @Operation(
      *     tags={"Entries"},
      *     summary="Create an entry.",
-     *     @OA\Parameter(
-     *         name="url",
-     *         in="query",
```

Comment:
I think the right fix is to remove this `in="query"` as suggested by @j0k3r in https://github.com/wallabag/wallabag/issues/7746#issuecomment-2454063764
the endpoints are not taking a JSON but normal POST

Reply:
Hi @yguedidi 

I tried that, but both are not working, did you have any other suggestions?

```xml
<args>
<item key="0">string</item>
<item key="1">
<![CDATA[ Missing required field "in" for @OA\Parameter(name="url") in \Wallabag\ApiBundle\Controller\EntryRestController->postEntriesAction() in /Users/eindex/repos/wallabag/src/Wallabag/ApiBundle/Controller/EntryRestController.php ]]>
</item>
</args>
```

<img width="1291" alt="image" src="https://github.com/user-attachments/assets/dc7ef8be-2902-482c-b990-fe0678aad585">

```xml
<args>
<item key="0">string</item>
<item key="1">
<![CDATA[ @OA\Parameter(name="url",in="body")->in "body" is invalid, expecting "query", "header", "path", "cookie" in \Wallabag\ApiBundle\Controller\EntryRestController->postEntriesAction() in /Users/eindex/repos/wallabag/src/Wallabag/ApiBundle/Controller/EntryRestController.php ]]>
</item>
</args>
```

<img width="1438" alt="image" src="https://github.com/user-attachments/assets/56c01e63-1fbb-403e-a086-2babbc121757">

Reply:
not an expert on OpenAPI, but quickly checking the doc and it seems we should replace `@OA\Parameter` by [`@OA\RequestBody`](https://zircote.github.io/swagger-php/reference/annotations.html#requestbody)